src/mono/wasm/runtime/assets.ts

```diff
@@ -38,6 +38,19 @@ const skipBufferByAssetTypes: {
     "dotnetwasm": true,
 };
 
+const skipInSnapshotByAssetTypes: {
```

Comment:
does this suppress loading or delay it? looking at the code below, these bools determine whether assets go into a 'before' or 'after' list, so 'skipInSnapshot' is probably the wrong name for it. `loadAfterSnapshotByAssetTypes` maybe?

Reply:
good feedback, renamed to `containedInSnapshotByAssetTypes` and collections to `alwaysLoadedAssets` and `containedInSnapshotAssets`